oak_utils/Cargo.toml

```diff
@@ -20,6 +20,7 @@ proc-macro2 = "*"
 # toolchains.
 tonic-build = { version = "*", default-features = false, features = [
   "prost",
+  "rustfmt",
```

Comment:
I was hesitant to add this back because @ebeasant-arm was having issues compiling on Arm where `rustfmt` is not available. Is it possible to make this optional?

Reply:
Happy to work around it  - there are things in the pipeline that will solve the original issue: No objections for this going back.

Reply:
Yeah, this was draft because I wanted to find out what would break &ndash; thanks for the pointer ARM-wards. (I also wonder if it will be another thing for #969).

Probably best to just drop this if it's going to cause inconvenience.